e2e/tests/localAuthorityMaintainsCaseAfterGatekeeping_test.js

```diff
@@ -0,0 +1,103 @@
+const config = require('../config.js');
+const recipients = require('../fixtures/recipients.js');
+
+let caseId;
+
+Feature('Case maintenance after gatekeeping');
+
+Before(async (I, caseViewPage, submitApplicationEventPage, sendCaseToGatekeeperEventPage) => {
+  if (!caseId) {
+    await I.logInAndCreateCase(config.swanseaLocalAuthorityEmailUserOne, config.localAuthorityPassword);
+    await I.enterMandatoryFields();
+    await caseViewPage.goToNewActions(config.applicationActions.submitCase);
+    submitApplicationEventPage.giveConsent();
+    await I.completeEvent('Submit');
+
+    // eslint-disable-next-line require-atomic-updates
+    caseId = await I.grabTextFrom('.heading-h1');
+    console.log(`Case ${caseId} has been submitted`);
+
+    I.signOut();
+
+    //hmcts login and send to gatekeeper
+    await I.signIn(config.hmctsAdminEmail, config.hmctsAdminPassword);
+    await I.navigateToCaseDetails(caseId);
+    caseViewPage.goToNewActions(config.administrationActions.sendToGatekeeper);
+    sendCaseToGatekeeperEventPage.enterEmail();
+    await I.completeEvent('Save and continue');
+    I.seeEventSubmissionConfirmation(config.administrationActions.sendToGatekeeper);
+    I.signOut();
+
+    await I.signIn(config.swanseaLocalAuthorityEmailUserOne, config.localAuthorityPassword);
+  }
+  await I.navigateToCaseDetails(caseId);
+});
+
+Scenario('local authority uploads documents', async (I, caseViewPage, uploadDocumentsEventPage) => {
```

Comment:
As I can imagine we are already doing similar tests, maybe we could just reuse the steps instead of duplicating them? What do you think?

Reply:
Would be good, we are just learning how to do the testing at the moment.